server

```diff
@@ -30,6 +31,7 @@ end
 
 post '/username' do
   cookies[:username] = params[:username]
+  cookies[:rem_votes] = 3
```

Comment:
the number of votes should be a property for the board, if we add it to the cookies here it'll be shared between boards and won't be renewed as long as the user didn't change his username

try having the number of votes in the board form then store it in the board hash then every time a user vote we'll check if he exceeded the max number of votes